src/content/docs/data-apis/custom-data/custom-events/data-requirements-limits-custom-event-data.mdx

```diff
@@ -82,35 +82,28 @@ When reporting custom events and attributes, follow these general requirements f
         * For events sent via [Event API](/docs/insights/insights-data-sources/custom-data/send-custom-events-event-api): 4096 characters
         * For events sent via agents: 255 characters
 
-          Maximum total attributes per event: 254. **Exception:** If you use an APM agent API, the max is 64.
+        Maximum total attributes per event: 254. **Exception:** If you use an APM agent API, the max is 64.
 
-          Maximum total attributes per event type: 48,000.
+        Maximum total attributes per event type: 48,000.
 
-          <Callout variant="important">
-            Charts may only display the first 255 characters of attribute values. For complete attribute values, use the [JSON chart type](/docs/insights/use-insights-ui/manage-dashboards/chart-types#widget-json) or [Query API](/docs/insights/insights-api/get-data/query-insights-event-data-api).
-          </Callout>
+        <Callout variant="important">
+          Charts may only display the first 255 characters of attribute values. For complete attribute values, use the [JSON chart type](/docs/insights/use-insights-ui/manage-dashboards/chart-types#widget-json) or [Query API](/docs/insights/insights-api/get-data/query-insights-event-data-api).
+        </Callout>
       </td>
     </tr>
 
     <tr>
       <td>
-        Naming syntax
+        Naming syntax for events and attributes
       </td>
 
       <td>
-        Attribute names can be a combination of alphanumeric characters, colons (`:`), periods (`.`), and underscores (`_`).
+        Can be a combination of alphanumeric characters, colons (`:`), periods (`.`), and underscores (`_`). We recommend starting the name with an alphabetical letter, and not using spaces: if you start with a non-letter character, or use a space, you'll need to put backticks around the name when querying. For more on querying non-standard names, see [NRQL syntax](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-how-nrql-works/#syntax).
```

Comment:
The rule doesn't entirely make sense.  We say names can be alphanumerics, colons, periods, and underscores and then talk about what to do if you use spaces.  Can you use spaces or not?  It's not clear.

NRDB allows names that don't fit these rules.  I can't swear that all systems are able to handle any given nonstandard name, though, so I recommend not changing the substance of the rules here.  Can we leave them the way they were and just update the backtick instructions to say something like "If a name begins with something other than a letter, or includes nonstandard characters, enclose the name with backticks in your NRQL query."  I think it's good to keep the example as well.